lib/players/computer.rb

```diff
@@ -0,0 +1,36 @@
+class Computer < Player
+  
+  def move(board)
+    available_moves = (1..9).select { |number| !board.taken?(number) } 
+    return win_or_stop_win(board).to_s if win_or_stop_win(board)
+    return best_moves(board, available_moves).to_s
+  end
+
+  def win_or_stop_win(board)
+    marker = self.token
+    opponent = ['X', 'O']
+    opponent.delete(marker)
+    Game::WIN_COMBINATIONS.each do |combo|
```

Comment:
One thing I didn't notice with including winning and blocking opponent winning in the same method, is that it causes the computer to make the wrong decision in a certain scenario.  If the computer can win on the next move, but the opponent can also win on the next move, it will choose to block the opponent from winning if the opponent's winning combo comes first in the array before the computer's winning combo.  Putting them in their own methods fixes this, because then it will only check for a winner first, and then proceed to blocking if no win is available.